site/www/style.css

```diff
@@ -3,6 +3,7 @@ html {
   font-size: 100%;
   line-height: 1.15;
   -webkit-text-size-adjust: 100%;
+  background: #fff;
```

Comment:
by specifying background at top level and inheriting, we can specify background color once (this will be useful for future dark mode)